File: tasks/prac6_python/task2.py
number = int(input("Введите число: "))

for i in range(1, 11):
    if i == 5:
        continue
    result = number * i
    print(f"{number} × {i} = {result}")

# https://pythonworld.ru/osnovy/cikly-for-i-while-operatory-break-i-continue-volshebnoe-slovo-else.html

File: tasks/prac6_js/task2.js
const prompt = require('prompt-sync')();

const number = parseInt(prompt("Введите число: "));

for (let i = 1; i <= 10; i++) {
    if (i === 5) {
        continue;
    }
    const result = number * i;
    console.log(`${number} × ${i} = ${result}`);
}